src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -58,6 +58,7 @@
     <InternalsVisibleTo Include="Microsoft.Diagnostics.Monitoring.ConfigurationSchema" />
     <InternalsVisibleTo Include="Microsoft.Diagnostics.Monitoring.OpenApiGen" />
     <InternalsVisibleTo Include="Microsoft.Diagnostics.Monitoring.Tool.UnitTests" />
+    <InternalsVisibleTo Include="Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests" />
```

Comment:
This should not be necessary. The FunctionalTests project should be running dotnet-monitor using the MonitorRunner class, not loading dotnet-monitor types into the test host.